lib/service_extensions.dart

```diff
@@ -109,15 +109,15 @@ const slowAnimations = ServiceExtensionDescription<num>(
 const String didSendFirstFrameEvent = 'ext.flutter.didSendFirstFrameEvent';
 
 // If you add a service extension to this file whose state should be restored on
-// page refresh/initial startup, add the extension to this list.
-const List<String> statefulExtensionsWhitelist = [
-  'ext.flutter.debugAllowBanner',
-  'ext.flutter.debugPaint',
-  'ext.flutter.debugPaintBaselinesEnabled',
-  'ext.flutter.inspector.show',
-  'ext.flutter.platformOverride',
-  'ext.flutter.profileWidgetBuilds',
-  'ext.flutter.repaintRainbow',
-  'ext.flutter.showPerformanceOverlay',
-  'ext.flutter.timeDilation',
-];
+// page refresh/initial startup, add the extension to this whitelist.
+final Map<String, ServiceExtensionDescription> toggleableExtensionsWhitelist = {
```

Comment:
you can reduce the boilerplate further
_toggleableExtensionsWhitelist = {};
for (var description in descriptions) {
  _toggleableExtensionsWhitelist[description.extension] = extension;
}

Reply:
Are you suggesting I keep a global list of ServiceExtensionDescriptions then?